lib/logSupport.py

```diff
@@ -40,7 +40,7 @@
 
 # Create a placeholder for a global logger (logging.Logger),
 # individual modules can create their own loggers if necessary
-log = None
+log: logging.Logger = None
```

Comment:
Could be also structlog.BoundLogger if using structlog (i.e. all the times structlog is installed)